packages/verifiable-credentials/src/verification/nftStatus.ts

```diff
@@ -0,0 +1,47 @@
+import { JsonRpcProvider, StaticJsonRpcProvider } from "@ethersproject/providers";
```

Comment:
nit on the file naming, u have the word "service" in the file name for the `VerifiableCredentialSignatureService`, but not in this one

probable can just make a services folder, then put both of these services there without "service" in the file name